manim/cli/render/commands.py

```diff
@@ -136,4 +138,40 @@ def __repr__(self):
             except Exception:
                 console.print_exception()
 
+    if config.notify_outdated_version:
+        manim_info_url = "https://pypi.org/pypi/manim/json"
+        warn_prompt = "Cannot check if latest release of manim is installed"
+        req_info = {}
+
+        try:
+            req_info = requests.get(manim_info_url, timeout=3)
+            req_info.raise_for_status()
+        except requests.exceptions.HTTPError:
+            logger.warning(f"HTTP Error: {warn_prompt}")
+            raise SystemExit
+        except requests.exceptions.ConnectionError:
+            logger.warning(f"Connection Error: {warn_prompt}")
+            raise SystemExit
+        except requests.exceptions.Timeout:
+            logger.warning(f"Timed Out: {warn_prompt}")
+            raise SystemExit
+        except Exception:
+            logger.warning(f"Something went wrong: {warn_prompt}")
+            raise SystemExit
+
+        try:
+            releases = req_info.json()["releases"].keys()
+
+            if max(releases) != __version__:
```

Comment:
looking at https://pypi.org/pypi/manim/json it seems it is better to directly pick version from info rather than doing this

```suggestion
        try:
            releases = req_info.json()["info"]["version"]

            if releases != __version__:
```